main.go

```diff
@@ -62,6 +64,19 @@ var podCommand = &cobra.Command{
 	},
 }
 
+var buildCommand = &cobra.Command{
+	Use: "build",
+	Run: func(cm *cobra.Command, args []string) {
+		fmt.Println("Building image from docker file at source")
+		err := cmd.BuildDockerFile(envVar, deployVar)
+		if err == nil {
+			fmt.Println("BuildConfig and ImageStream Created Successfully, Build can now be Started")
+		} else {
+			fmt.Println(err)
```

Comment:
we can either log errors or fmt.Errof("This is an error: %s",err )

Reply:
added log.fatal